src/components/organisms/Dashboard.tsx

```diff
@@ -68,13 +70,28 @@ const Dashboard: React.FC = () => {
                                             mappingFunction={dataSourceMappings[item.datasourceId].mapping}
                                             query={dataSourceMappings[item.datasourceId].query}
                                         >
-                                            {(data) => (
-                                                <ThresholdChart
-                                                    data={data}
-                                                    height={parent.height}
-                                                    width={parent.width}
-                                                />
-                                            )}
+                                            {(data) => {
+                                                switch (item.visualisationType) {
+                                                    case VisualisationType.LINE:
+                                                        return (
+                                                            <LineChart
+                                                                width={parent.width}
+                                                                height={parent.height}
+                                                                data={data}
+                                                            />
+                                                        );
+                                                    case VisualisationType.THRESHOLD:
+                                                        return (
+                                                            <ThresholdChart
+                                                                data={data}
+                                                                height={parent.height}
+                                                                width={parent.width}
+                                                            />
+                                                        );
+                                                    default:
+                                                        return <Text>Visualisation currently not supported.</Text>;
```

Comment:
Translate to norwegian :)

Reply:
Yep. Please request changes when you encounter an error.

Reply:
I.e. don't "comment".